jobs/config.json

```diff
@@ -10090,6 +10090,32 @@
       "UNKNOWN"
     ]
   },
+  "pull-cri-containerd-build": {
+    "args": [
+      "./test/e2e_node/build.sh"
+    ],
+    "scenario": "execute",
+    "sigOwners": [
+      "sig-node"
+    ]
+  },
+  "pull-cri-containerd-node-e2e": {
+    "args": [
+      "--deployment=node",
+      "--gcp-project=cri-c8d-pr-node-e2e",
+      "--gcp-zone=us-central1-f",
+      "--node-args=--image-config-file=test/e2e_node/image-config.yaml --node-envs=PULL_REFS",
```

Comment:
cwd will be `cri-containerd` rather than kubernetes, probably you want to use the test infra version of the image config?